bin/check_golint

```diff
@@ -42,7 +42,6 @@ MESSAGES_RECONFIGURE=()
 
 check_uncleaned_package "github.com/mysteriumnetwork/node/identity" 6
 check_uncleaned_package "github.com/mysteriumnetwork/node/tequilapi" 11
-check_uncleaned_package "github.com/mysteriumnetwork/node/session" 6
```

Comment:
Extra karma points for cleaning up packages

Reply:
So where is my approval?